tools/DevDiagnostics/DevHome.DevDiagnostics/Helpers/ImageOptionsInternalTool.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using DevHome.DevDiagnostics.Models;
+using DevHome.DevDiagnostics.Properties;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Media;
+
+namespace DevHome.DevDiagnostics.Helpers;
+
+public class ImageOptionsInternalTool : Tool
+{
+    private const string ButtonText = "\ue9d5";
+    private static readonly string _toolName = CommonHelper.GetLocalizedString("ImageOptionsName");
+
+    public ImageOptionsInternalTool()
+        : base(_toolName, ToolType.Unknown, Settings.Default.IsImageOptionsToolPinned)
+    {
+    }
+
+    public override IconElement GetIcon()
+    {
+        return new FontIcon
+        {
+            Glyph = ButtonText,
+            FontFamily = (FontFamily)Application.Current.Resources["SymbolThemeFontFamily"],
+        };
+    }
+
+    internal override void InvokeTool(ToolLaunchOptions options)
+    {
+        if (TargetAppData.Instance.TargetProcess == null || TargetAppData.Instance.TargetProcess.MainModule == null)
+        {
+            return;
+        }
+
+        try
+        {
+            FileInfo fileInfo = new FileInfo(Environment.ProcessPath ?? string.Empty);
+            Directory.SetCurrentDirectory(fileInfo.DirectoryName ?? string.Empty);
+
+            var startInfo = new ProcessStartInfo()
+            {
+                FileName = "DevHome.IfeoTool.exe",
+                Arguments = TargetAppData.Instance.TargetProcess.MainModule.ModuleName,
```

Comment:
doesn't this crash on elevated processes?

Reply:
Yeah, but I don't think the solution is locale specific.  The real problem is that TargetAppData.Instance.TargetProcess is implemented in a way that crashes if you try to access something with higher elevation.  I don't think the fix should be scoped to my change, as we likely have other places that don't work.  We should instead fix TargetAppData to avoid using that code I think.

Reply:
https://github.com/microsoft/devhome/issues/3848

Reply:
we should handle the exception here like we do in other pages and prompt the user to elevate